pyproject.toml

```diff
@@ -7,6 +7,35 @@ requires = [
     "tomli; python_version < '3.11'",
 ]
 
+[project]
+name = "pyogrio"
+dynamic = ["version"]
+authors = [{ name = "Brendan C. Ward", email = "bcward@astutespruce.com" }]
+maintainers = [{ name = "pyogrio contributors" }]
```

Comment:
Followed the pattern we have in other packages keeping the author field but also adding a more general "maintainers" description